hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -507,6 +507,20 @@ private FlinkOptions() {
       .defaultValue(4) // default WRITE_TASKS * COMPACTION_DELTA_COMMITS * 0.2 (assumes 5 commits generate one bucket)
       .withDescription("Parallelism of tasks that do actual compaction, default is 4");
 
+  public static final ConfigOption<String> COMPACTION_SEQUENCE = ConfigOptions
+      .key("compaction.sequence")
+      .stringType()
+      .defaultValue("LIFO") // default WRITE_TASKS * COMPACTION_DELTA_COMMITS * 0.2 (assumes 5 commits generate one bucket)
```

Comment:
LIFO is not recommended. It's possible the oldest plan never get executed and the active timeline get longer and longer.